pkg/cli/cli.go

```diff
@@ -395,35 +388,35 @@ func cvdCommands(opts *subCommandOpts) []*cobra.Command {
 }
 
 func connectionCommands(opts *subCommandOpts) []*cobra.Command {
-	connectFlags := &ConnectFlags{opts.RootFlags, ""}
+	connFlags := &ConnectFlags{opts.RootFlags, "", false /*skipConfirmation*/}
```

Comment:
nit: let's use `field: value` declaration to initialize here, I think it still could be in one line.